projects/plugins/super-cache/changelog/fix-preserve-file-permissions-on-update

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Preserves original permissions of a file after updating it.
```

Comment:
The patch is good. I just have a tiny suggestion. The changelog file is what compiles into the changelog on the plugin release. 

![image](https://user-images.githubusercontent.com/3737780/210935691-6c48a7a3-e9ed-4bb5-8bcd-f4f9a209e3d1.png)

The above screenshot is the current changelog. I think writing something like:
> Fixed an issue causing wp-config.php file permission to change

Reply:
Thank you - I wasn't aware of that. Fixed with eebf97a.